cc-statistics/README.md

```diff
@@ -0,0 +1,236 @@
+# Common Crawl statistics notebooks
+
+### Introduction
+
+The goal of this repo is to allow users to easily and interactively view statistics about the Common Crawl data set.
```

Comment:
Maybe more precise: "... view statistics about the Common Crawl webgraph data sets and interactively explore the graphs."

Also the directory "cc-statistics" should be renamed.